packages/react/plugins/component-testing/index.ts

```diff
@@ -45,7 +45,19 @@ import {
 export function nxComponentTestingPreset(
   pathToConfig: string,
   options?: NxComponentTestingOptions
-) {
+): {
+  specPattern: string;
+  devServer: {
+    framework?: 'react';
+    bundler?: 'vite' | 'webpack';
+    viteConfig?: string;
```

Comment:
viteConfig is the resolved config not the path to the config. which is why we were getting the cryptic errors about React not being installed.

since cypress will auto load the cypress.config.ts file in the project root we actually don't need to provide this value. so it can be omitted.